ai/main.py

```
@@ -38,21 +43,48 @@ def synopsys(book_id: int, authorization: str = Header(None)):
     try:
         book_info = get_book(book_id, authorization)
-        story = get_all_story(book_id, authorization)
-        if book_info and story:
+
+        if book_info:
             genre = book_info.get("genre")
             character = book_info.get("character")
             background = book_info.get("background")
-            result = ai.generate_story(genre, character, background, story, choice)
+            result = ai.generate_story(
+                genre = genre,
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)